pants.toml

```diff
@@ -96,14 +96,11 @@ root_patterns = [
   "/",
 ]
 
-[environments-preview.platforms_to_local_environment]
-linux_arm64 = "//:default_env"
-linux_x86_64 = "//:default_env"
-# TODO(#7735): Set to `//:macos_local_env` after figuring out why our Build Wheels Mac job is
+[environments-preview.aliases]
+default_env = "//:default_env"
```

Comment:
@stuhood until we add `--local-override`, you will now in pants.rc want to set this:

```toml
[environments-preview.aliases]
default_env = "//:macos_local_env"
```

Reply:
Ideally, we have three environments:

* macOS local, which avoids PATH
* linux local, which is default of PATH and Pyenv
* CI, which is only PATH

For now, we only define:

* macOS local
* default env, which includes both PATH and PYENV

It sounds like you personally have been using a pants.rc file to avoid PATH being in your config. To do that override now, you would need to redefine `[environments-preview.aliases]`. 

Instead, I'm proposing that in a followup, `--aliases` can define several platforms for the same platform,, then the `--override` option lets you choose which `__local__` should point to. I intentionally am saving that for a (low priority) dedicated PR so we can confirm the desirability of that.